src/packages/client/.eslintrc.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {
```

Comment:
Just a suggestion: what we do in our monorepo is that we have a base eslintrc and we require it in each package to add additional rules (and eventually you can remove it entirely when the code is uniform across packages).

Reply:
Thanks for the suggestion I just did that 😃!